### ISSUE TITLE
"False Positives" in "local resources checker"  (error in checker)

### ISSUE THREAD

**Issue by gernotstarke** [aim42/htmlSanityCheck#41]
for example, checking the "Griffon Guide" reports the following false-positives

```
Missing Local Resources Check
157 anchor tag href attribute checked, 157 missing local resources found.

    local resource "api/griffon/core/mvc/MVCGroupManager.html" missing
    local resource "api/griffon/core/injection/Module.html" missing 
```
